src/transformers/cloudflare.test.ts

```diff
@@ -0,0 +1,39 @@
+import { assertEquals } from "https://deno.land/std@0.172.0/testing/asserts.ts";
+import { ParsedUrl } from "../types.ts";
+import { CloudflareParams, parse, transform } from "./cloudflare.ts";
+
+const img =
+  "https://assets.example.comeight=64,f=auto,background=red/uploads/avatar1.jpg"
```

Comment:
I think something's wriong with this URL!

Reply:
Fixed